app/assets/stylesheets/profiles.scss

```diff
@@ -0,0 +1,21 @@
+// Place all the styles related to the profiles controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.profile-page {
+  width: 100%;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+}
+
+.cover-placeholder {
+  background: 50% no-repeat #fafafa;
+  background-size: cover;
```

Comment:
Properties should be ordered background, background-color, background-size, border-radius, height, margin-bottom, width